android/app/build.gradle

```diff
@@ -23,9 +23,17 @@ android {
     }
     buildTypes {
         debug {
+            buildConfigField("String", 'OAUTH_BASE_URL', '"http://10.0.2.2:8080/auth/"')
+            buildConfigField("String", 'OAUTH_CIENT_ID', '"fhircore-client"')
+            buildConfigField("String", 'OAUTH_CLIENT_SECRET', '"74383j43u4y3b3jj"')
+
             testCoverageEnabled true
         }
         release {
+            buildConfigField("String", 'OAUTH_BASE_URL', '"http://10.0.2.2:8080/auth/"')
+            buildConfigField("String", 'OAUTH_CIENT_ID', '"fhircore-client"')
```

Comment:
Move these values to the `local.properties` and implement reading as in this file https://github.com/opensrp/opensrp-client-reveal/blob/master/opensrp-reveal/build.gradle#L93